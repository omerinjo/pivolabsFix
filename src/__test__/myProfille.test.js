import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store'
import { BrowserRouter as Router } from 'react-router-dom';

import MyProfile from '../screens/myProfile/profile'

const initialState = {};

const mockStore = configureStore();
let store;

beforeEach(() => {
    //creates the store with any initial state or middleware needed  
    store = mockStore(initialState)
})


describe('<MyProfile> rendering', () => {
    test('render correctly', () => {
        const component = renderer.create(<Router><MyProfile store={store} /> </Router>)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})