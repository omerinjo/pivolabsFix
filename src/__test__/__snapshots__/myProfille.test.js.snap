// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyProfile> rendering render correctly 1`] = `
Array [
  <div
    className="container"
  >
    <nav
      className="navbar-container navbar navbar-expand-lg navbar-light bg-light"
      data-test="guestNavigation"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          href="/login"
          onClick={[Function]}
        >
          Login 
        </a>
        <a
          className="nav-link"
          href="/standing"
          onClick={[Function]}
        >
          Standing 
        </a>
      </div>
    </nav>
    <div
      className="jumbotron"
    >
      <div
        className="container"
        style={
          Object {
            "height": "50vh",
          }
        }
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <h2>
              Name
            </h2>
            <h2>
              Email
            </h2>
            <h2>
              My likes
            </h2>
          </div>
          <div
            className="col"
          >
            <h2>
              
            </h2>
            <h2>
              
            </h2>
            <h2>
              
            </h2>
          </div>
        </div>
      </div>
      <button
        className="btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Change password
      </button>
    </div>
  </div>,
  " ",
]
`;
